File: INICIANTE/1072/main.c
#include <stdio.h>

int main() {
    int n, x, inInterval = 0;
    scanf("%d", &n);
    for (int times = 0; times < n; times++) {
        scanf("%d", &x);
        if (0 <= x && x <= 20) inInterval++;
    }
    printf("%d in\n", inInterval);
    printf("%d out\n", n - inInterval);
    return 0;
}

File: INICIANTE/1072/main.js
let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let inInterval = 0;
let n = parseInt(lines.shift());
for (let times = 0; times < n; times++) {
    let x = parseInt(lines.shift());
    if (0 <= x && x <= 20) inInterval++;
}
console.log(`${inInterval} in`);
console.log(`${n - inInterval} out`);
